knowage-core/src/main/java/it/eng/spagobi/api/AbstractDocumentResource.java

```diff
@@ -133,7 +133,7 @@ public Response updateDocument(String label, String body) {
 		return Response.ok().build();
 	}
 
-	public Response deleteDocument(@PathParam("label") String label) {
+	protected Response deleteDocument(@PathParam("label") String label) {
```

Comment:
If possible, I would prefer private but this is an abstract class, we need a check.

Reply:
This one can't be private, it is being used in DocumentResource.java (two files, one of them is v2).